src/actions/event-actions.js

```diff
@@ -85,6 +85,83 @@ export const getEvents = (term = null, page = 1, perPage = 10, order = 'id', ord
         if (extraFilters.allows_publishing_dates_filter) {
```

Comment:
@tomrndom please move all this to its own method ( like parseFilters) as we are doing on other grid like attendees and tickets and also apply to CSV counter part